src/languages/spark/spark.formatter.ts

```diff
@@ -80,54 +81,23 @@ const reservedCommands = [
   'WINDOW',
 ];
 
-const reservedBinaryCommands = [
-  // set booleans
-  'INTERSECT',
-  'INTERSECT ALL',
-  'INTERSECT DISTINCT',
-  'UNION',
-  'UNION ALL',
-  'UNION DISTINCT',
-  'EXCEPT',
-  'EXCEPT ALL',
-  'EXCEPT DISTINCT',
-  'MINUS',
-  'MINUS ALL',
-  'MINUS DISTINCT',
-  // apply
-  'CROSS APPLY',
-  'OUTER APPLY',
```

Comment:
Spark doesn't actually support `APPLY`, just like it also doesn't support `MINUS`. Likely just a result of copy-pasting these here without checking when Spark dialect was created.